hadoop-plugin-test/src/main/gradle/positive/embeddedFlow.gradle

```diff
@@ -0,0 +1,69 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath files("${project.pluginTestDir}/hadoop-plugin-${project.version}.jar", "${project.pluginTestDir}/hadoop-plugin-${project.version}-SNAPSHOT.jar")
+    // Must manually specify snakeyaml for testing, not required for users
+    classpath 'org.yaml:snakeyaml:1.18'
```

Comment:
Have you confirmed that this version of snakeyaml is in LinkedIn artifactory? The build might fail internally if it's not there.

Reply:
Good thinking! Yes, I see `snakeyaml-1.18.jar` in artifactory :+1: